.prettierrc

```diff
@@ -1 +0,0 @@
-proseWrap: always
```

Comment:
@alexeagle Was this option intentionally added or just copied from another file? If we include this, then prettier likes to wrap yaml keys without adding the proper `>-` directives.

Reply:
it's copy-pasta - it's useful for markdown though.